### ISSUE TITLE
Your instructions don't say how to set the settings for those with 12 GB of video memory or more 

### ISSUE THREAD

**Issue by abomonov** [lllyasviel/Fooocus#2397]
**Read Troubleshoot**

[x] I confirm that I have read the [Troubleshoot](https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md) guide before making this issue.
Yes

**Describe the problem**
You do not have in the instructions where to write --always-high-vram to use video memory to the maximum, I have 12 GB of video memory and consumes only 6 GB. Where do I put --always-high-vram to maximize the capabilities? 

**Full Console Log**
Your instructions don't say how to set the settings for those with 12 GB of video memory or more.  

Thanks, Fooocus the best!

**Comment by mashb1t** [comment]
Thanks for the hint. Depending on your GPU you can not only use `--always-high-vram`, but `--always-gpu` instead.
Arguments can be added after entry_with_update.py depending on OS either in the run.bat or after entering the command.

**Comment by abomonov** [comment]
Thank you! I took your advice and tested on windows and rtx 3060 12 GB video card. When I set the value in run_realistic after entry_with_update.py --always-high-vram the video memory was used 11 GB instead of 6 GB. When I set --always-gpu video memory started to be used at maximum up to 11.8 GB and generation slowed down ten times. The conclusion is that in my case I need to use --always-high-vram then the generation speed increases and everything works fine.
